src/aten/UnaryOps.cpp

```diff
@@ -1,31 +1,118 @@
 #include <ATen/ScalarOps.h>
 #include <ATen/XPUNativeFunctions.h>
 #include <ATen/core/Tensor.h>
+#include <ATen/native/Resize.h>
 #include <ATen/native/TensorIterator.h>
-#include <torch/library.h>
-
 #include <aten/sycl/UnaryKernels.h>
 #include <aten/sycl/UnaryLogKernels.h>
+#include <torch/library.h>
 
 namespace at {
 
+template <typename Stub>
```

Comment:
Can we implement a lambda to align the API as `stub`. The lambda is a wrapper to route to our `abs_kernel`. Like,
```
auto xpu_abs_stub = [](DeviceType dev, TensorIteratorBase iter) { abs_kernel(iter) };
```

Reply:
Dismiss. `unary_op_impl_with_complex_to_float_out` is static function in PyTorch.